Propagated environment from server process to session. This is useful with
auth-none=1 and per-user instances.

diff --git a/src/cpp/server/ServerSessionManager.cpp b/src/cpp/server/ServerSessionManager.cpp
index e7e6af38c9..3d991752a2 100644
--- a/src/cpp/server/ServerSessionManager.cpp
+++ b/src/cpp/server/ServerSessionManager.cpp
@@ -121,6 +121,8 @@ core::system::ProcessConfig sessionProcessConfig(
 
    // pass our uid to instruct rsession to limit rpc clients to us and itself
    core::system::Options environment;
+   // Copy existing environment
+   core::system::environment(&environment);
    uid_t uid = core::system::user::currentUserIdentity().userId;
    environment.push_back(std::make_pair(
                            kRStudioLimitRpcClientUid,
